src/astar/shiden.js

```diff
@@ -50,7 +50,8 @@ const scheduleTask = async ({
         { Recurring: { frequency: TASK_FREQUENCY, nextExecutionTime } },
         // { Fixed: { executionTimes: [0] } },
         shibuyaHelper.config.paraId,
-        0,
+        paraTokenIdOnTuring,
+        { V1: { parents: 1, interior: { X1: { Parachain: shibuyaHelper.config.paraId } } } },
         encodedCallData,
         encodedCallWeight,
     );
```

Comment:
Don’t you need to apply the same change to remove the below line? @imstar15 
```
    const taskViaProxy = turingHelper.api.tx.proxy.proxy(turingAddress, 'Any', taskExtrinsic);
```

Reply:
Okay. Repalced scheduleXcmpTask with scheduleXcmpTaskThroughTask in shiden.